composer/callbacks/checkpoint_saver.py

```diff
@@ -542,3 +545,8 @@ def _rotate_checkpoints(self, sharding_enabled: bool = False):
             else:
                 if dist.get_global_rank() == 0:
                     shutil.rmtree(prefix_dir)
+
+    def _log_to_mosaicml_logger(self, state: State, logger: Logger):
+        for destination in logger.destinations:
+            if isinstance(destination, MosaicMLLogger):
+                destination.log_checkpoint_upload_start(state)
```

Comment:
Can we instead log the information directly and make the name of this fn say its only logging save time?